scalameta/contrib/shared/src/main/scala/scala/meta/contrib/implicits/ReplaceExtensions.scala

```diff
@@ -0,0 +1,12 @@
+package scala.meta.contrib.implicits
+
+import scala.collection.immutable.Seq
+import scala.meta.contrib._
+
+trait ReplaceExtensions {
+  implicit class ExtensionReplacers[A](a: A) {
+    def replace[B](bs: Seq[B])(implicit ev: Replace[A, B]): A = ev.replace(a, bs)
```

Comment:
I'm ok with replace but names are important so I'll throw out a few more. Here are my thoughts on various words.

Set: () => B. I don't care what (if anything) comes in, give me a B.  
Replace:  _: A => B. Take A and Replace with B without inspecting the value A.  B is decided upon without referring to the specific A, but I am only going to put a B in if an A was there before.
Map/Transform: A => B: Take A and based on it's contents, create a B.